rest-api/app_util.py

```diff
@@ -18,13 +18,25 @@
 _GMT = pytz.timezone('GMT')
 SCOPE = 'https://www.googleapis.com/auth/userinfo.email'
 
+def handle_database_disconnect(err):
+  """Intended to catch DBAPIError's thrown during a request cycle and transform them into 503's.
+  If the DBAPIError does not represent an invalidated connection, reraise the error.
+
+  Usage: app.register_error_handler(DBAPIError, handle_database_disconnect)
+  """
+  if err.connection_invalidated:
```

Comment:
I think its logged automatically isn't it?  Or maybe not.  If not, we should be able to just change [this line](https://github.com/all-of-us/raw-data-repository/blob/e27183644b270664adbba94fdbd315a82f3f1bdb/rest-api/main.py#L41) to have a type tuple: 

```
if isinstance(exception, (HTTPException, DBAPIError)):
    # logging, etc
```

I guess it would be safest to just add the error in - if we're double logging we can always remove it, right? And double logging is better than no logging?